Lib/idlelib/idle_test/test_run.py

```diff
@@ -72,7 +72,7 @@ class PseudeInputFilesTest(unittest.TestCase):
 
     def test_misc(self):
         shell = MockShell()
-        f = run.PseudoInputFile(shell, 'stdin', 'utf-8')
+        f = run.StdInFile(shell, 'stdin', 'utf-8')
```

Comment:
Changing the file class names is fine, but I would prefer StdinFile, StdoutFile or even StdInputFile, StdOutputFile. StdInFile and StdOutFile seem too dense with capitals.

With 'utf-8' properly made the default, we can remove the repeated and now redundant passing of the same.  I will make and push both changes if you prefer.